src/config/permissions.php

```diff
@@ -149,4 +149,15 @@
     */
 
     'migration_sub_folder' => '',
+    
+    /*
+    |--------------------------------------------------------------------------
+    | Run seeder in production env
+    |--------------------------------------------------------------------------
+    |
+    | if this true, the seed commands in PermissionSeedCommand can be run in the production env using the force option
```

Comment:
**Important**

we should make the default value to be true. also in the description we may then write something like:

"Force seeding package seeder without interruption by questions in terminal. The default value is true so, if you like to stop forcing the seeder to run just change the value to false"